Harvest.Web/ClientApp/src/Projects/ProjectDetailContainer.tsx

```diff
@@ -25,14 +25,19 @@ export const ProjectDetailContainer = () => {
   const [newFiles, setNewFiles] = useState<BlobFile[]>([]);
 
   const [notification, setNotification] = usePromiseNotification();
+  const [beyondCloseoutDisplayDate, setBeyondCloseoutDisplayDate] = useState(false);
 
   useEffect(() => {
     // get rates so we can load up all expense types and info
     const cb = async () => {
       const response = await fetch(`/Project/Get/${projectId}`);
 
       if (response.ok) {
-        setProject(await response.json());
+        const project = await response.json() as Project;
+        setProject(project);
+
+        //show Closeout link if we are in last week of or beyond project end 
+        setBeyondCloseoutDisplayDate(new Date().getTime() - new Date(project.end).getTime() >= (7 * millisecondsPerDay));
```

Comment:
This calculation seems handy enough to use a few times and probably will come in handy again.  Perhaps instead you can move this to the Calculation Utils itself as a function like `getDaysDiff(date1, date2)` and then here it would be easier to read that you are just checking to see if the date diff is >= 7.